3rdparty/workspace.bzl

```diff
@@ -19,7 +19,17 @@ def _jar_artifact_impl(ctx):
         )
         srcjar_attr ='\n    srcjar = ":%s",' % src_name
 
-    build_file_contents = """
+    if ctx.attr.lang == "kotlin":
```

Comment:
To use the `filegroup` here like with scala, `kt_jvm_import` would need handle filegroups. Or `kt_jvm_library` would need to handle not having any sources. IIUC @hsyed 

Perhaps we could only spit out this extra skylark/starlark if there's at least one kotlin library in dependencies.yaml.